Config.xcconfig

```diff
@@ -1,2 +1,6 @@
 MARKETING_VERSION = 2.7.0
 CURRENT_PROJECT_VERSION = 45
+
+// Source for the base of all bundle IDs
+// -> Intents extensions and app will derive bundleID from here
+PRODUCT_BUNDLE_IDENTIFIER = com.sindresorhus.Actions
```

Comment:
Not sure why this is needed. The bundle identifier will never change, so doesn't really make sense to optimize for reuse.

Reply:
I can't run on a real device with an existing bundle ID and changing only the app id will lead to a compilation error because the extension ids arent prefixed with it. This allows a contributor to change all ids at once